Source/WebGPU/WebGPU.xcodeproj/xcshareddata/xcschemes/wgslfuzz.xcscheme

```diff
@@ -17,7 +17,7 @@
                BlueprintIdentifier = "1CA5B4F02A6F28C400E5F297"
                BuildableName = "wgslfuzz"
                BlueprintName = "wgslfuzz"
-               ReferencedContainer = "container:WebGPU.xcodeproj">
+               ReferencedContainer = "container:../../../../../../../WebKit/OpenSource/Source/WebGPU/WebGPU.xcodeproj">
```

Comment:
Was this intentional?

Reply:
@smfr guessing this was not intentional

Reply:
No, but there is something funky about the project settings for wgslfuzz.xcscheme, which always gets touched on my filesystem.

Reply:
Oh thanks, I will file a bug Monday to investigate that issue

Reply:
https://bugs.webkit.org/show_bug.cgi?id=269734